### ISSUE TITLE
SmartTemplates 4.9: %identity(name)% result lower case only

### ISSUE THREAD

**Issue by MarkyMarkDE** [RealRaven2000/SmartTemplate4#336]
`%identity(name)%` results the right name but if you use e.g. CamelCase it returns all in lower case.
So it do not read quite well from identity settings.

Example: Identity name is "MyName Blubb" returns "myname Blubb"

**Comment by RealRaven2000** [comment]
Can I close this issue? I cannot reproduce this, using the following settings in Tb ESR140

<img width="434" height="231" alt="Image" src="https://github.com/user-attachments/assets/a2721ccb-30c6-4471-9b9a-841e3ed2d789" />

```html
<p>
    Test identity case (issue 336): %identity(name)%
</p>
```

The result uses the display name with the existing case:

<img width="540" height="120" alt="Image" src="https://github.com/user-attachments/assets/f0c15989-cd5e-4a1f-bf15-997fc97ac39f" />

**Comment by MarkyMarkDE** [comment]
give me little time to check it with my support template for this project here

**Comment by MarkyMarkDE** [comment]
yes, you can close this, the first option in extended settings "Capitalize names" was the cause of my problem.

<img width="818" height="266" alt="Image" src="https://github.com/user-attachments/assets/1ea44ddc-244a-4802-bc31-1603c77b6b0a" />
--

see the path behind "kind regards," this is now the right display output and before it was "smarttemplates Support Germany"
<img width="1028" height="265" alt="Image" src="https://github.com/user-attachments/assets/b197f832-6a95-49e1-b8dc-0d8f7686b016" />

maybe we should set by default "Capitalize Names" default to false if is not

OR

we make an exception rule: if the user set "Capitalize Names" to `true` he get a sub option where he is able to set "ignore above rule on CamelCase Names" or something like this.

**Comment by RealRaven2000** [comment]
related: #305
